frame/oracle/src/lib.rs

```diff
@@ -347,6 +348,8 @@ pub mod pallet {
 		RewardingAdjustment(T::Moment),
 		/// Answer from oracle removed for staleness. \[oracle_address, price\]
 		AnswerPruned(T::AccountId, T::PriceValue),
+		/// Price changed by oracle \[asset_id, price\]
```

Comment:
why the "\" in comments?

Reply:
Just following previous code convention. I assume it's escape the `[` for some reason.

Reply:
rustdoc tries to read `[content]` as a link and removes the brackets @blasrodri